src/stp/new_plays/GenericPass.cpp

```diff
@@ -41,21 +41,21 @@ GenericPass::GenericPass() : Play() {
                                                                                  std::make_unique<role::Halt>(role::Halt("halt_7"))};
 }
 
-uint8_t GenericPass::score(world_new::World* world) noexcept { return 50; }
+uint8_t GenericPass::score(world_new::World* world) noexcept { return 10; }
```

Comment:
Class constant variable...

Reply:
No because the scoring will be fixed in a future PR